test/ci_integration.sh

```diff
@@ -54,20 +55,21 @@ setup_environment() {
 }
 
 main() {
+  export SUITE="${SUITE:-}"
+
   set -eu
   # Setup trap handler to auto-cleanup
   export TMPDIR="${DELETE_AT_EXIT}"
   trap finish EXIT
 
-  # Setup environment
+  # Setup environment variables
   setup_environment
   set -x
+
   # Execute the test lifecycle
-  bundle install
-  bundle exec kitchen create
-  bundle exec kitchen converge
-  bundle exec kitchen converge
-  bundle exec kitchen verify
+  kitchen create "$SUITE"
+  kitchen converge "$SUITE"
```

Comment:
Same question as with `destroy` here. Are we certain this runs to completion, repeatedly?

Reply:
AFAIK bug was fixed